### ISSUE TITLE
Whether to consider adding column levels

### ISSUE THREAD

**Issue by javaht** [AbsaOSS/spline-spark-agent#742]
Whether to consider adding column levels  ？

**Comment by wajda** [comment]
What do you mean?

**Comment by javaht** [comment]
Column level data lineage

**Comment by wajda** [comment]
Yes, but how exactly would you like it to be represented? What output is missing from the agent?

**Comment by wajda** [comment]
The output JSON contains attributes definitions. Each attribute has a `childRef` property that refers to either another attribute (the one it depends on and derives the value from), or an expression that was used to calculate the values for the given attribute. This forms the attribute (or column) level lineage.
